src/impl/dispatch.js

```diff
@@ -1,17 +1,42 @@
 // @flow
 import { RingBuffer, ring } from './buffers';
 
+const TASK_BATCH_SIZE: number = 1024;
```

Comment:
I'm interested to know the implications of this on performance and collaboration with the other js on the page.

My gut says this is a pretty heavy performance increase!